wear/src/main/java/io/homeassistant/companion/android/home/views/EntityUi.kt

```diff
@@ -99,7 +99,7 @@ fun EntityUi(
                 }
             },
             enabled = isEnabled,
-            toggleControl = { ToggleSwitch(isChecked) },
+            toggleControl = { Switch(checked = isChecked) },
```

Comment:
This may work as a quick fix but visually it's back to M2 🤨
![image](https://github.com/home-assistant/android/assets/8148535/f3c894d2-5724-4369-a373-b1324ec5867a)

As we need the M2 ToggleChip (because of the background position stuff), maybe it is better to drop the toggle completely, result: matching what Google Home is doing but with a normal chip/button?

Reply:
Picking this back up since its [about time](https://github.com/home-assistant/android/pull/4622#issuecomment-2344235139) we did lol. Should we keep it as is for now and work on updating to the latest changes in another PR? Seems we are now 5 alphas behind and Google has again made lots of changes in the most recent one last week lol.

https://developer.android.com/jetpack/androidx/releases/wear-compose#1.0.0-alpha24

I am not sure on dropping the toggle as currently it more closely resembles default dashboard which has toggles :)

Reply:
Yes we should really get back to being recent (but why is there still nothing beta or stable?!). I no longer like the Google Home toggles as much so merging this with M2 toggles so it works, catching up, and then seeing what to do sounds reasonable to me :)

Reply:
yea we did decide to use an alpha release so sometimes things like this is just inevitable lol